### PR TITLE
Feature/version five improvements

### PR DESCRIPTION
**Pull request by jmdrawneek**

### CODE REVIEW

File: src/scss/base/_links.scss [hunk 1]
```diff
@@ -48,3 +67,18 @@ nav {
     }
   }
 }
+
+.styled-link--external {
+  @extend %styled-link;
+
+  &::after {
+    @extend %rc-icon;
+    @extend .rc-icon-external--small--dark;
```
Comment (mbbillz) [inline src/scss/base/_links.scss]: Shall we agree that this is the way to use icons as pseudo elements? I've noticed that this is done in several ways in other areas of the code. e.g @extend .rc-icon-*::after or sometimes it's not nested. Just for consistency's sake :).

Comment (jmdrawneek) [reply src/scss/base/_links.scss]: Yea definitely. Suprised this works actually. I'll have to double check we've got the utility class generating at what level. Can't remember the reasoning for assigning icons to pseudo-elements.

Comment (mbbillz) [reply src/scss/base/_links.scss]: https://codepen.io/anon/pen/rpgmgy - Yeah me too.. but it seems there are three ways of doing the same thing, second one is most readable I think.

**Comment by mbbillz** [comment]
All looks good to me.

**Comment by jmdrawneek** [comment]
@mbbillz Did you mean to close it?

File: src/scss/base/_links.scss (hunk 1 above)
Comment (jmdrawneek) [reply src/scss/base/_links.scss]: Yep agreed. Raise a card about this and we'll do a sweep unifying the style.

**Pull request merged**